src/mozillavpn.cpp

```diff
@@ -1106,6 +1111,13 @@ void MozillaVPN::deactivate() {
   scheduleTask(new TaskControllerAction(TaskControllerAction::eDeactivate));
 }
 
+void MozillaVPN::silentSwitch() {
+  logger.log() << "VPN tunnel silent server switch";
+
+  deleteTasks();
```

Comment:
Why deleting all the tasks?

Reply:
I was worried that multiple switch tasks might be in the queue. After a successful switch we do not want to switch again if not necessary.
If multiple pings in a row indicate an unstable connection the task will be added multiple times. Or am I missing something?